packages/create-llama/questions.ts

```diff
@@ -125,6 +131,26 @@ export const askQuestions = async (
     return; // early return - no further questions needed for community projects
   }
 
+  if (program.template === "llamapack") {
+    const availableLlamaPacks = await getAvailableLlamapackExamples();
+    const { llamapack } = await prompts(
+      {
+        type: "select",
+        name: "llamapack",
+        message: "Select llama pack example",
+        choices: availableLlamaPacks.map((pack) => ({
+          title: pack.name,
+          value: pack.file_path,
+        })),
+        initial: 0,
+      },
+      handlers,
+    );
+    program.llamapack = llamapack;
+    preferences.llamapack = llamapack;
+    return; // early return - no further questions needed for llamapack projects
```

Comment:
@thucpn can we provide the postinstall action to install dependencies for llama pack projects?

Reply:
The user should also then get some information what do to next? E.g. calling `poetry run python example.py`